### DISCUSSION TITLE
Facing Problems the reports module in old UI

### DISCUSSION THREAD

**Discussion by rishabhsuriwtc** [egovernments/Digit-Core#53] [DIGIT Platform]
Since reports is only available in old UI, i am trying to add more columns in the report. the extra columns are invisible on the table which is shown in the screen and also invisible in the downloaded PDF. in the downloaded excel, i am able to see all the columns. is there any workaround for this ?

**Comment by pradeepkumarcm-egov** [comment]
Check whether new columns are added under [sourceColumns](https://github.com/egovernments/configs/blob/9c67d05a85769a23f6a3dbf5d11bc9b612aadb54/reports/config/rainmaker-pgr-reports.yml#L55).

Make sure **showColumn: false** flag is not added for newly added [fields](https://github.com/egovernments/configs/blob/9c67d05a85769a23f6a3dbf5d11bc9b612aadb54/reports/config/rainmaker-pgr-reports.yml#L85).
 
Also, check whether data is present in the report column with a horizontal scroll. Please share the report yml file and output screens for further analysis.

**Reply by VaibhavJain1996** [reply]
[New PGR ULB report.txt](https://github.com/egovernments/Digit-Core/files/11229306/New.PGR.ULB.report.txt)
![PGR ULB search](https://user-images.githubusercontent.com/101560545/231955106-81678bf8-851d-40b2-af76-3dcc2836cb8c.jpg)
[PGR ULBReport PDF.pdf](https://github.com/egovernments/Digit-Core/files/11229308/PGR.ULBReport.PDF.pdf)
[PGR ULBReport.xlsx](https://github.com/egovernments/Digit-Core/files/11229309/PGR.ULBReport.xlsx)
![ULB Report Search Result](https://user-images.githubusercontent.com/101560545/231955121-b48098bb-70df-4882-8712-cad51fe82a46.jpg)



Hi Pradeep,
I have attached the screenshots of output screens, pdf of report, and excel sheet of report. In excel sheet, we are able to see all the fields but not in pdf. Also, I have checked that new columns are added under sourceColumns and showColumn: False flag is not adding for newly added fields. Please help us further over the mentioned issue.

Thank you so much for your support!

**Reply by pradeepkumarcm-egov** [reply]
Vaibhav, the report query and output format looks correct. Check the report response. The query result returning all the values but because of css issue, the data not printed in the UI. Add the horizontal scroll option for the report or rearrange the last 4 columns in the beginning(for verification).
